lib/stat_tracker.rb

```diff
@@ -310,4 +310,46 @@ def lowest_scoring_home_team
     end
     lowest_home_team_name
   end
+
+  def least_accurate_team(season)
+    season_stats = @all_data_hash[:game_teams].select do |game|
```

Comment:
Maybe this association of `game_id` to season can be made into a helper method later on.

Reply:
100% percent agree. Good idea.